web/themes/custom/pineapple/sass/layouts/nodes/_author.scss

```diff
@@ -0,0 +1,64 @@
+.author-body {
+  .cover-image {
+    height: 250px;
+    background-repeat: no-repeat;
+    background-position: 50%;
+
+    @include breakpoint($lap) {
+      height: 350px;
+    }
+  }
+
+  .author-container {
+    margin-bottom: $unit *4;
+    padding: $unit $unit*3;
+    padding-left: 0;
+    background-color: $light-gray;
+
+    @include breakpoint($lap) {
+      display: flex;
+      flex-direction: row;
+    }
+
+    .author-info-container {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+
+      @include breakpoint($lap) {
+        flex-direction: row;
+      }
+
+      .profile-image {
+        width: 50%;
+        max-width: 290px;
+        margin-top: -30%;
+        z-index: 1;
+
+        @include breakpoint($lap) {
+          width: 200%;
+          margin-top: -15%;
+        }
+
+        @include breakpoint($desk) {
+          width: 100%;
+        }
+
+        img {
+          border-radius: 50%;
+          width: 100%;
+          object-fit: cover;
+        }
+      }
+
+      .author-text {
+        margin-left: $unit *3;
```

Comment:
The use of this margin creates some inconsistency, i suggest you remove the padding left 0 on line 15 of this file and the add a margin right on the image container to push your text, because right now you're adding space to right through padding and then adding space to the left through margin

Reply:
You mean text container?

Reply:
done